uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Test/Fabricator/Relationship.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class CRM_HRCore_Test_Fabricator_Relationship {
+
+  private static $defaultParams = [
+    'sequential' => 1,
+    'name_a_b' => "test AB",
+    'name_b_a' => "test BA",
+    'contact_type_a' => "Individual",
+    'contact_type_b' => "Individual",
+  ];
+
+  public static function fabricateRelationshipType($params = []) {
+    $params = array_merge(self::$defaultParams, $params);
+
+    $result = civicrm_api3(
+      'Activity',
```

Comment:
Is this correct? If it's not, it looks like your tests didn't catch this and we definitely need more tests (specially negative tests) to the importers